src/style.css

```diff
@@ -1 +1,114 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Book.ttf') format('truetype');
+  font-weight: 300, 400;
```

Comment:
The `font-weight` property should not have multiple values like `300, 400`. Each `@font-face` rule should specify a single `font-weight`. Consider splitting this into separate `@font-face` rules for each weight.